.github/workflows/build-and-test.yml

```diff
@@ -68,28 +68,28 @@ jobs:
       - name: Create all API docs for artifacts/website and all reference docs
         run: ./bin/test-documentation
 
-  # documentation-tests:
-  #   name: Run documentation tests
-  #   runs-on: ubuntu-20.04
-  #   steps:
-  #     - name: Checkout
-  #       uses: actions/checkout@v2
-  #       with: # https://github.com/olafurpg/setup-scala#faster-checkout-of-big-repos
-  #         fetch-depth: 100
-
-  #     - name: Fetch tags
-  #       run: git fetch --depth=100 origin +refs/tags/*:refs/tags/*
-
-  #     - name: Set up JDK 8
-  #       uses: coursier/setup-action@v1.1.2
-  #       with:
-  #         jvm: adopt:8 # need jdk8 to build docs
-
-  #     - name: Cache Coursier cache
-  #       uses: coursier/cache-action@v6.3
-
-  #     - name: Create all API docs for artifacts/website and all reference docs
-  #       run: ./bin/test-docs-code-only
+  documentation-tests:
+    name: Run documentation tests
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with: # https://github.com/olafurpg/setup-scala#faster-checkout-of-big-repos
+          fetch-depth: 100
+
+      - name: Fetch tags
+        run: git fetch --depth=100 origin +refs/tags/*:refs/tags/*
+
+      - name: Set up JDK 11
+        uses: coursier/setup-action@v1.1.2
+        with:
+          jvm: adopt:11 # need jdk11
```

Comment:
it seems that the tests in the docs used to be compiled with jdk11
https://github.com/lagom/lagom/blob/c4e9511c058b985e19f9b1216cda8a3ba5725e18/.travis.yml#L73-L75

So, I will just keep it as such. 🤷